A critical error has ocurred while running the game!
Error type: <class 'TypeError'>
What happened: load() takes no keyword arguments
Where it happened:  file 'D:\Documentos\Programas\C\nundertale\main.py', at line 20Why it happned: Bad coding xD

Full error traceback:
 Traceback (most recent call last):
  File "D:\Documentos\Programas\C\nundertale\main.py", line 20, in main
    initial_lore_level = GameLevel(lore_sprites)
  File "D:\Documentos\Programas\C\nundertale\model\GameLevel.py", line 11, in __init__
    self.load_level_sprites()
  File "D:\Documentos\Programas\C\nundertale\model\GameLevel.py", line 20, in load_level_sprites
    self.sprite_list.append(GameSprite(raw, name))
  File "D:\Documentos\Programas\C\nundertale\model\GameSprite.py", line 6, in __init__
    self.loaded_image = image.load(raw, namehint=name)
TypeError: load() takes no keyword arguments
